### DISCUSSION TITLE
Websocket using bun server - Unable to access request object in ctx.extras

### DISCUSSION THREAD

**Discussion by ashbuilds** [enisdenjo/graphql-ws#521] [Q&A] [answered]
Hi, Im unable to access request object in `makeHandler` api, are there some other config am I missing here?

**Screenshot**

<img width="769" alt="Screenshot 2023-11-16 at 12 16 39" src="https://github.com/enisdenjo/graphql-ws/assets/2686288/911ec399-1c11-4ad2-a888-c39e90975751">

<img width="916" alt="Screenshot 2023-11-16 at 12 15 09" src="https://github.com/enisdenjo/graphql-ws/assets/2686288/cbfbffd1-35bc-4dce-b7b5-1dee95d12759">
<img width="505" alt="Screenshot 2023-11-16 at 12 15 30" src="https://github.com/enisdenjo/graphql-ws/assets/2686288/f9a00966-c659-4b1b-ad2d-70ccf3e91d75">

**Expected Behaviour**
As described in examples we should be able to access request object in context.

**Comment by enisdenjo** [comment, marked as answer]
There is no request in the context extra when using Bun (see docs). [It's simply not available because Bun doesn't provide it.](https://bun.sh/docs/api/websockets)
